plugin/ctrlsf.vim

```diff
@@ -139,6 +139,7 @@ let s:default_mapping = {
     \ "open"    : ["<CR>", "o"],
     \ "openb"   : "O",
     \ "split"   : "<C-O>",
+    \ "vsplit"  : "<C-o>",
```

Comment:
ok.i map to v for vertical split and h for horizontal split. dont really use visual mode in there...